.ci/macos-steps.yml

```diff
@@ -20,10 +23,24 @@ steps:
     brew install ccache pkg-config arpack eigen glpk hdf5 json-c lapack lzo nlopt snappy xz
   displayName: Install dependencies
 
-- task: CMake@1
-  displayName: Setup
-  inputs:
-    cmakeArgs: -DBUILD_EXAMPLES=OFF -DBUNDLE_JSON=OFF -DBUNDLE_NLOPT=OFF -DENABLE_TESTING=ON -DENABLE_COVERAGE=OFF -DBUILD_META_EXAMPLES=OFF ..
+- ${{ if eq(parameters.use_mkl, 'true') }}:
```

Comment:
@vigsterkr how do you capture env variables here? $(use_mkl) doesn't seem to do anything...